src/bindings/python/src/pyopenvino/graph/model.cpp

```diff
@@ -750,6 +761,57 @@ void regclass_graph_Model(py::module m) {
                 :rtype: int
              )");
 
+    model.def(
+        "get_sink_index",
+        [](ov::Model& self, const ov::Output<ov::Node>& value) -> int64_t {
```

Comment:
Could you please also add overload for `(ov::Model& self, const ov::Output<const ov::Node>& value)`

Reply:
Sure, I've added the overload now.